src/libs/TransactionRecoveryApi/AxelarRecoveryApi.ts

```diff
@@ -11,14 +11,21 @@ import { BigNumber } from "ethers";
 import { throwIfInvalidChainIds } from "../../utils";
 
 export enum GMPStatus {
-  SRC_GATEWAY_CALLED = "source_gateway_called",
-  DEST_GATEWAY_APPROVED = "destination_gateway_approved",
-  DEST_EXECUTED = "destination_executed",
-  DEST_EXECUTE_ERROR = "destination_execute_error",
+  SRC_GATEWAY_CALLED = "called",
+  DEST_GATEWAY_APPROVED = "approved",
+  DEST_EXECUTED = "executed",
```

Comment:
let's  not change these... the src/dest fields make it clearer where things are executed (and btw... these were specifically requested)

Reply:
@canhtrinh oh okay, I didn't realize that those were specifically requested. Btw, should we prefix `source` and `dest` to the additions as well?

Reply:
imo, they can stay as is i think